architecture/smg/02-smg-design.md

```diff
@@ -54,22 +55,28 @@ ssh ubuntu@<internal_server_ip>
 ```
 
 A better way is to add SSH configuration to your SSH config file to allow single step
-access. An example configuration for the same access is:
+access. An example configuration for the same access (typically stored at `~/.ssh/config`) is:
 
 ```bash
-Host bastion
+Host datalabs-bastion-1
         HostName <bastion_public_ip>
         User deploy
         IdentityFile ~/.ssh/<private_key>
         ForwardAgent yes
 
-Host k8s-master
+Host datalabs-k8s-master
         HostName <internal_ip>
         User deploy
         ProxyCommand ssh bastion -W %h:%p
```

Comment:
I think `bastion` needs to be updated to `datalabs-bastion-1` here

Reply:
Agreed, good spot.